mappings/net/minecraft/test/class_4512.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_4512 net/minecraft/test/class_4512
```

Comment:
?

Reply:
Had to move this to avoid access level errors. I'll try to find a name.

Reply:
Hmm, just removing this mapping isn't a good fix.

Reply:
It turns out I was wrong, it's not package-private after all. The package private class was a different one I found a name for.